validator/service/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "mc-validator-service"
+version = "1.0.0"
+authors = ["MobileCoin"]
+edition = "2018"
+license = "GPL-3.0"
+
+[[bin]]
```

Comment:
btw is this intentional?

Reply:
What are you referring to? This follows a pattern we've done in other services where we place most code in a reusable library and separately from that provide a binary that puts it all together.